package.json

```diff
@@ -7,6 +7,13 @@
   "umd:main": "dist/index.js",
   "module": "dist/index.module.js",
   "jsnext:main": "dist/index.module.js",
+  "exports": {
```

Comment:
Let's add `.` as well like we do in [preact](https://github.com/preactjs/preact/blob/master/package.json#L13)

Reply:
Good call, updated that 👍